src/main/java/com/lob/model/Letter.java

```diff
@@ -236,6 +239,10 @@ public ZonedDateTime getSendDate() {
         return sendDate;
     }
 
+    public LetterSize getLetterSize() {
+        return letterSize;
+    }

```

Comment:
Does this return `null` when the size field doesn't exist?

Reply:
yes